packages/react-aria-components/src/Tabs.tsx

```diff
@@ -100,7 +105,8 @@ export interface TabPanelRenderProps {
    * `shouldForceMount` prop is true, and the corresponding tab is not selected.
    * @selector [inert]
    */
-  isInert: boolean
+  isInert: boolean,
+  state: TabListState<unknown>
```

Comment:
can we do something like
```
  state: Omit<TabListState<unknown>, 'collection'>
```

Reply:
or, since that's in reverse of how we construct the new state object, we could also do something like
```
interface SubsetState<T = unknown> {
  disabledKeys: TabListState<T>.disabledKeys
  ... repeat, same way we construct the actual object
}
```

Reply:
The state used for TabPanel and TabList is the entire state, so I don't think we need to change the type here.

Reply:
Maybe it just needs a comment? I didn't realize this wasn't a subset
<img width="1004" alt="Screenshot 2023-06-28 at 2 04 32 PM" src="https://github.com/adobe/react-spectrum/assets/698229/929b2747-901b-4c50-bff5-540f9f487a32">

Reply:
Thanks, looks like I was missing comments for those. Just added.